src/main/java/org/folio/dew/batch/bursarfeesfines/AccountReader.java

```diff
@@ -1,5 +1,8 @@
 package org.folio.dew.batch.bursarfeesfines;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
```

Comment:
These used?

Reply:
I don't think so. I'm not sure why these imports appear.